src/components/checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import './styles.scss'
+
+const Checkbox = ({ id, value, onChange }) => (
+    <div>
```

Comment:
Her kan du vere skikkelig kul og prøve deg med ein `React.Fragment` i staden for `div`